### DISCUSSION TITLE
Cloud Rendering Or WebGL?

### DISCUSSION THREAD

**Discussion by ICpeng** [o3de/o3de#8186] [General]
Will o3de support cloud rendering Or WebGL ?

**Comment by ValorZard** [comment]
A webgl2 export would be really nice for embedding games into sites like itch.io or new grounds. This is especially important for people who want to make games for portfolio pieces so that they can easily show them off. Also, some people may not want to download a game and just want to easily try it out in the browser.

**Comment by lmbr-pip** [comment]
O3DE is still getting a roadmap together, but you can see items on it here: https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Froadmap+

Right now neither WebGL or cloud rendering feature as public roadmap items.

I would see WebGl and cloud rendering as two different things. It would probably be interesting to identify what your use cases are, what problems do you need to solve and how may these help.

You can also propose making feature suggestions. IMHO its best to work through one of the special interest groups (in this case sig-graphics-audio) to help shape and idea or to find potential contributors in this area.

**Comment by moudgils** [comment]
WebGPU is the way to go forward in this space although this also comes with some big challenges. Ignoring the need for a new RHI backend and support for a new shader pipeline we would also need to work on reducing app size for O3de in order for it to be viable.

**Comment by DarklinuxFr** [comment]
why especially in the cloud? rendering with a cluster would be more relevant and in this case proposed O3ED as basic software in major Linux distributions
